scripts/scripts.js

```diff
@@ -389,13 +389,40 @@ async function addPrevNextLinksToArticles() {
   heroLinkContainer.append(nextLink);
 }
 
+const scanAllTextNodes = (element) => {
+  const allChildNodes = element.childNodes;
+  allChildNodes.forEach((oNode) => {
+    if (oNode.nodeType !== Node.TEXT_NODE) {
+      scanAllTextNodes(oNode);
+      return;
+    }
+    if (oNode.nodeValue.trim() === '') {
+      return;
+    }
+    if (oNode.nodeValue !== '# # #') {
+      return;
+    }
+    const span = document.createElement('span');
+    span.classList.add('article-end-divider');
+    span.textContent = oNode.nodeValue;
+    element.replaceChild(span, oNode);
+  });
+};
+
+const centerArticleDivider = (articleSections) => {
+  articleSections.forEach((article) => {
+    scanAllTextNodes(article);
+  });
+};
+
 function annotateArticleSections() {
   const template = getMetadata('template');
   if (template !== 'Article') {
     return;
   }
   const articleSections = document.querySelectorAll('main > .section');
 
+  centerArticleDivider(articleSections);
```

Comment:
1. can this be done after the blocks are rendered in loadLazy? since anyways it is at the bottom of the page. no need to do such heavy calculations up front.
2. we can further reduce the scope by looking only at main > .section that is not .abstract or .hero-container and only look inside that element for scanning the text nodes.

![Screenshot 2023-09-27 at 6 26 49 PM](https://github.com/hlxsites/accenture-newsroom/assets/7074782/90c53c29-a73f-4335-8995-4481d61d78c6)

Reply:
@amol-anand I moved the function to loadLazy and add specific query for the default-content-wrapper. Please review once again. Thank you

Reply:
Thanks for making this change!